registrations/test_tasks.py

```diff
@@ -297,21 +297,24 @@ def test_is_registered_on_whatsapp_true(self):
         def cb(request):
             data = json.loads(request.body)
             self.assertEqual(
-                data,
-                {"number": "+27820000000", "msisdns": ["+27821111111"], "wait": True},
+                data, {"blocking": "wait", "contacts": ["+27821111112"]},
             )
-            return (200, {}, json.dumps([{"status": "valid", "wa_id": "27821111111"}]))
+            return (200, {}, json.dumps([{"contacts": [{"input": "+27821111112",
+                                                        "status": "valid",
+                                                        "wa_id": "27821111112"}]
+                                          }]))
```

Comment:
The Engage API returns an object, not an array, eg. `{"contacts": [...]}` not `[{"contacts": [...]}]`